schemas/region.schema.json

```
@@ -0,0 +1,133 @@
+          "type": "object",
+          "properties": {
+            "lat": {
+              "description": "Latitude coordinate.",
+              "type": "number",
+              "minimum": -90,
+              "maximum": 90
+            },
+            "lng": {
+              "description": "Longitude coordinate.",
+              "type": "number",
+              "minimum": -180,
+              "maximum": 180
+            },
+            "address": {
+              "description": "City name of this location.",
+              "type": "string"
+            }
+          },
+          "required": [ "lat", "lng", "address" ]
+        },
+        "polygon": {
+
+        }
+      },
+      "required": [ "defaultLocation", "polygon" ]
+    },
+    "currency": {
+      "description": "Details of the currency used in this region.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The ISO 4217 currency code of this currency.",
+          "type": "string"
+        },
+        "symbol": {
+          "description": "The symbol of this currency for the specified locale. For example, for the US Dollar, the symbol is '$'' if the specified locale is the US, while for other locales it may be 'US$'",
+          "type": "string"
+        },
+        "format": {
+          "description": "Format details of the currency.",
+          "type": "object",
+          "properties": {
+            "unit": {
+              "description": "The main unit pattern of this currency. Example: '£0'",
+              "type": "string"
+            },
+            "subunit": {
+              "description": "The subunit pattern of this currency if any. Example: '£0.00'",
+              "type": "string"
+            }
+          },
+          "required": [ "unit" ]
+        }
+      },
+      "required" : [ "format" ]
+    }, 
+    "measurementSystem": {
+      "description": "Measurement system to use in this region.",
+      "enum": ["metric", "imperial"],
+      "default": "metric"
+    }, 
+    "trafficHand": {
+      "description": "Traffic side on the road to keep to. 'lht' for left , 'rht' for right.",
+      "enum": ["lht", "rht"],
+      "default": "rht"      
+    },
+    "servers": {
+      "description": "Servers to allocate this region.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "minItems": 1,
+      "uniqueItems": true
+    },
+    "eligibility": {
+      "description": "This region eligibility.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "minItems": 1,
+      "uniqueItems": true
+    },
+    "feeds": {
+      "description": "Set of public transport feeds for this region",
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "minItems": 1,
+      "uniqueItems": true
+    }
+  },
+  "required": [ "code", "timezone", "locale", "coverage", "currency", "servers" , "feeds" ]
```

There might not be public transport in the area, which means `feeds` can be optional.

```suggestion
  "required": [ "code", "timezone", "locale", "coverage", "currency" ]
```